config/database.yml

```diff
@@ -1,29 +1,31 @@
-development: &development
-  adapter: mysql2
+<% db_adapter = ENV["SIGNONOTRON2_DB_ADAPTER"].nil? ? "mysql" : ENV["SIGNONOTRON2_DB_ADAPTER"] %>
```

Comment:
This could be tidied up a little by fetching `ENV.fetch("SIGNONOTRON2_DB_ADAPTER", "mysql")`